eZ/Publish/Core/MVC/Symfony/Routing/ChainRouter.php

```diff
@@ -1,24 +1,15 @@
 <?php
 
 /**
- * File containing the ChainRouter class.
- *
  * @copyright Copyright (C) eZ Systems AS. All rights reserved.
  * @license For full copyright and license information view LICENSE file distributed with this source code.
  */
+declare(strict_types=1);
+
 namespace eZ\Publish\Core\MVC\Symfony\Routing;
 
 use Symfony\Cmf\Component\Routing\ChainRouter as BaseChainRouter;
 
 class ChainRouter extends BaseChainRouter
 {
-    public function generate($name, $parameters = [], $referenceType = self::ABSOLUTE_PATH)
-    {
-        if ($name instanceof RouteReference) {
-            $parameters += $name->getParams();
-            $name = $name->getRoute();
-        }
```

Comment:
It doesn't work anymore. In theory we could provide support for syntax

```php
$this->urlGenerator->generate('', [
    RouteObjectInterface::ROUTE_OBJECT => $routeRef
])
```
but it will require to impl.  `\Symfony\Cmf\Component\Routing\ChainedRouterInterface` which seems to be overkill for me.